src/emulator/modules/SceNgs/SceNgs.cpp

```diff
@@ -41,8 +43,10 @@ EXPORT(int, sceNgsPatchRemoveRouting) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceNgsRackGetRequiredMemorySize) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceNgsRackGetRequiredMemorySize, uint32_t hSysHandle, void *unk, uint32_t *size) {
+    *size = 1;
+    LOG_WARN("Stubbed sceNgsRackGetRequiredMemorySize called (size = 1)");
```

Comment:
Don't use yet another way to do stubs, either make `stubbed()` or do `unimplemented_impl("sceNgsRackGetRequiredMemorySize  => size=1");`

Reply:
This got suggested in Discord. (LOG_WARN + return 0). I also find way bad to show an unimplemented warn for stubbed functions. They're different things.

Reply:
That's why I suggested to alternatively make a `stubbed("whatever")` logging func (and a macro `STUBBED()` as we do for `unimplemented`), to use for that purpose, if you feel like it.